£Lerror_reason¢DtypeEvalueYpHTTPSConnectionPool(host='archive.ics.uci.edu', port=443): Max retries exceeded with url: /ml/datasets.html?format=&task=other&att=cat&area=&numAtt=&numIns=100to1000&type=other&sort=attUp&view=list (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x7f1d2840ef10>, 'Connection to archive.ics.uci.edu timed out. (connect timeout=2)'))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı